### PR TITLE
Form fixed

### PR DESCRIPTION
**Pull request by Proladge**

### CODE REVIEW

**Comment by Proladge** [comment]
To describe in a few words.
@alexx97-python has done improvement to message that is shown on form validation.
And fixed table layout, by shortening super long links

File: project/dbmanager/templates/dbmanager/index.html [hunk 1]
```diff
@@ -135,13 +135,23 @@
...
             <a href="{{ item }}"> {{ item[:20] + '...' }}</a>
+            {% else %}
+            <a href="{{ item }}"> {{ item }}</a>g
             {% endif %}
             {% endfor %}
         </td>
         <td>{{ knowledge.additional_answers }}</td>
-        <td>{{ knowledge.additional_links }}</td>
+        <td>
+            {% for item in knowledge.splitedAddLinks %}
```
Comment (Proladge) [inline project/dbmanager/templates/dbmanager/index.html:147]: @alexx97-python  please rename ```knowledge.splitedAddLinks``` to be ```knowledge.splitedAdditionalLinks```

File: project/dbmanager/views.py [hunk 2]
```diff
@@ -13,16 +13,13 @@
...
-    return links.split(';')
-
-
 @dbmanager_blueprint.route('/')
 @token_required
 def index():
     all_data = knowledgeManager.get_all_data()
     for item in all_data:
-       item['splitedLinks'] = split_links(item['links'])
+        item['splitedLinks'] = item['links'].split(';')
```
Comment (Proladge) [inline project/dbmanager/views.py:21]: @alexx97-python  Does it work well? 
When we have no links and ```item['links']``` returns ```null``` or whatever

File: templates/base.html [hunk 3]
```diff
@@ -12,15 +12,7 @@
...
-{% if messages %}
-{% for message in messages %}
-<p>{{message}}</p>
-{% endfor %}
-{% endif %}
-{% endwith %}
-
-{% block body %}{% endblock %}
+{% block content %}{% endblock %}
 <footer>Developed by Dan Bilokha </footer>
```
Comment (danbilokha) [inline templates/base.html:16]: you can remove that :D

Comment (Proladge) [reply templates/base.html:16]: or simply add ```and Alex-python```   to it

File: project/dbmanager/templates/dbmanager/index.html (hunk 1 above)
Comment (alexx97-python) [reply project/dbmanager/templates/dbmanager/index.html:147]: @Proladge ok, I will change it

File: project/dbmanager/views.py (hunk 2 above)
Comment (alexx97-python) [reply project/dbmanager/views.py:21]: @Proladge It works perfect as for me, doesn't cause any error, just leave the block empty without links

**Pull request merged**